boards/arm/mps3/board.yml

```diff
@@ -5,3 +5,4 @@ board:
   - name: 'an547'
     variants:
     - name: 'ns'
+    - name: 'tsv'
```

Comment:
what is a `tsv` board variant ?
Please describe it, what / when is it useful to build for this board.

Remember to update board documentation: https://github.com/zephyrproject-rtos/zephyr/blob/ca38d4bdc0e25cdcf176098454de052dff7b9dbc/boards/arm/mps3/doc/index.rst

Reply:
I asked this question a week ago..

Based on the commiter’s email address, my guess would be that this is a shortened version of `Tsavorite` for `Tsavorite Scalable Intelligence`.

As far as I can see, this new board only differs from the base board on some of the console output and the memory layout/partitions used.
It seems to me this should be a downstream overlay instead of an extra board variant upstream.